x-pack/plugins/security_solution/cypress/e2e/detection_alert_details/navigation.cy.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { expandFirstAlert, waitForAlertsPanelToBeLoaded } from '../../tasks/alerts';
+import { createCustomRuleEnabled } from '../../tasks/api_calls/rules';
+import { cleanKibana } from '../../tasks/common';
+import { login, visit } from '../../tasks/login';
+
+import { getNewRule } from '../../objects/rule';
+import type { CustomRule } from '../../objects/rule';
+
+import { ALERTS_URL } from '../../urls/navigation';
+import {
+  ALERT_DETAILS_PAGE_BACK_TO_ALERTS,
+  OPEN_ALERT_DETAILS_PAGE_CONTEXT_MENU_BTN,
+  TIMELINE_CONTEXT_MENU_BTN,
+} from '../../screens/alerts';
+import { PAGE_TITLE } from '../../screens/common/page';
+import { OPEN_ALERT_DETAILS_PAGE } from '../../screens/alerts_details';
+
+describe('Alert Details Page Navigation', () => {
+  describe('navigating to alert details page', () => {
+    let rule: CustomRule;
+    before(() => {
+      rule = getNewRule();
+      cleanKibana();
+      login();
+      createCustomRuleEnabled(rule, 'rule1');
+      visit(ALERTS_URL);
+      waitForAlertsPanelToBeLoaded();
+    });
+
+    it('should navigate to the details page from the alert context menu', () => {
+      cy.get(TIMELINE_CONTEXT_MENU_BTN).first().click({ force: true });
+      // It opens in a new tab by default, for testing purposes, we want it to open in the same tab
+      cy.get(OPEN_ALERT_DETAILS_PAGE_CONTEXT_MENU_BTN)
+        .invoke('removeAttr', 'target')
+        .click({ force: true });
```

Comment:
NIT: Please wrap this into a task method.

Reply:
Ended up being removed